src/core/generateZodSchema.test.ts

```diff
@@ -168,6 +168,13 @@ describe("generateZodSchema", () => {
     );
   });
 
+  it("should generate a literal schema for a single union", () => {
+    const source = `export type Identity = | "superman";`;
```

Comment:
I was not even aware of this notation 😅 make totally sense!

Reply:
Haha ya I use it to remind myself this will be a union but haven't filled out everything yet. Typescript is pretty rad :P